WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift

```diff
@@ -258,16 +258,17 @@ class NoticeView: UIView {
     }
 
     private enum Metrics {
-        static let cornerRadius: CGFloat = 13.0
+        /// The corner radius of the background (colored) and its shadow.
+        static let cornerRadius: CGFloat = 4.0
         static let dualLayoutMargins = UIEdgeInsets(top: 6.0, left: 6.0, bottom: 6.0, right: 6.0)
         static let labelLineSpacing: CGFloat = 18.0
     }
 
     private enum Appearance {
-        static let actionBackgroundColor = UIColor.white.withAlphaComponent(0.5)
```

Comment:
What was the purpose of this and why was it removed?

Reply:
It was a constant and was only used in line 152: 

```swift
actionBackgroundView.backgroundColor = Appearance.actionBackgroundColor
```

But since the `backgroundColor` now uses `notice.style.backgroundColor`, it's no longer needed.